<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Catalog</class>
 <widget class="QDialog" name="Catalog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Каталог товаров</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxCat">
      <property name="title">
       <string>Каталог товаров:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineNumOfAgent">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineFIOOfAgent">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushCatPrint">
           <property name="maximumSize">
            <size>
             <width>111</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Печать таблицы...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushCatOrderHider">
           <property name="text">
            <string>Создать заказ &gt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableCatView">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Фильтр:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineFilter"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushCatAdd">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushCatEdit">
           <property name="text">
            <string>Редактировать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushCatDelete">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushCatDiscount">
           <property name="text">
            <string>Скидка...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBoxOrderer">
      <property name="title">
       <string>Накладная:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>№:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineNumOfOverhead">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Дата:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit">
           <property name="displayFormat">
            <string>dd.MM.yy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableCatOrderer"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Кол-во:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineNumOfOrders">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Баллов:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineAllPoints">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Всего:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineAllCash">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pushAddToTrash">
           <property name="text">
            <string>Добавить...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushDelFromTrash">
           <property name="text">
            <string>Удалить...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushTrashApply">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
